intellij-plugin/build.gradle.kts

```diff
@@ -219,6 +220,25 @@ dependencies {
   implementation(project(":apollo-tooling"))
   implementation(project(":apollo-normalized-cache-sqlite"))
   implementation(libs.sqlite.jdbc)
+  implementation(libs.apollo.runtime.published)
 }
 
 fun isSnapshotBuild() = System.getenv("COM_APOLLOGRAPHQL_IJ_PLUGIN_SNAPSHOT").toBoolean()
+
+apollo {
+  service("apolloDebug") {
+    packageName.set("com.apollographql.apollo3.debug")
+    schemaFile.set(file("../libraries/apollo-debug-server/src/androidMain/resources/schema.graphqls"))
+    introspection {
+      endpointUrl.set("http://localhost:12200/")
+      schemaFile.set(file("../libraries/apollo-debug-server/src/androidMain/resources/schema.graphqls"))
+    }
```

Comment:
Do we need introspection for this given that the schema is checked in source control?

Reply:
Not really, but it's handy because it allows the IJ plugin to be configured with a URL, then we can execute queries in-IDE.

Reply:
👀 oohoho I see 👍 . The `introspection {}` block is a bit misleading here, maybe we could rework that later... Anyways, cool stuff 👍